cbmgmtx/errors.go

```diff
@@ -42,3 +43,16 @@ func (e contextualError) Error() string {
 func (e contextualError) Unwrap() error {
 	return e.Cause
 }
+
+type ServerInvalidArgError struct {
+	argument string
```

Comment:
Does users need to be able to access these fields to create their own error messages?

Reply:
Made them public